samples/net/cloud/aws_mqtt/sample.yaml

```diff
@@ -0,0 +1,8 @@
+sample:
+  description: MQTT sample app to AWS IoT cloud
+  name: aws-mqtt
+tests:
+  sample.net.cloud.aws_mqtt:
+    harness: net
+    platform_whitelist: sam_e70_xplained
```

Comment:
Remove the whitelisting and put instead minimum ram requirement. That way the sample can be run any device with network connectivity and enough ram.